src/extension.ts

```diff
@@ -21,6 +21,14 @@ import { LeanTaskGutter, LeanTaskMessages } from './taskgutter';
 import { StaticServer } from './staticserver';
 import { LibraryNoteLinkProvider } from './librarynote';
 
+async function checkLean3(): Promise<boolean> {
+    const lean4 = extensions.getExtension('leanprover.lean4');
+    if (!lean4) {
+        return true;
+    }
+    return (await lean4.activate()).aborted;
```

Comment:
I really don't like the name `aborted` here, can we pick something more self-explanatory and concrete like `isLean3Project` or `didNotFindLean4` or ...?